components/eam/tools/topo_tool/cube_to_target/cube_to_target.F90

```diff
@@ -217,6 +256,8 @@ program convterr
   
   allocate ( target_corner_lon(ncorner,ntarget),stat=alloc_error)
   allocate ( target_corner_lat(ncorner,ntarget),stat=alloc_error)
+  allocate ( target_corner_lon_deg(ncorner,ntarget),stat=alloc_error)
```

Comment:
Are these just the lat/lon corner coords, but in degrees instead of radians? If so, do we really need to allocate new arrays for this?

Reply:
good point... these variables seem to just be straight copies of the other cell corner values. Must have been an oversight in the original, so I'll remove them and use the other variables.

Reply:
done.